dcli/generatorCli.py

```diff
@@ -107,10 +107,15 @@ def parse_target_method_args(self, args):
 
 
     def c(self):
-        """ Generates a basic Java challenge """
+        """ Generates a basic C challenge """
         answers = ask_usual()
         answers.update(askCQuestions())
 
+        language = C()
+
+        # append default value
+        answers[TARGET_METHOD_RETURN_VALUE] = language.get_default_value(answers['targetMethodReturn'])
```

Comment:
This is not enough to make it works.  
Take the example on the Java one, you also have to rebase I think.

Reply:
ok i'll check that, thanks